threenums.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```

Comment:
Good job, but, again, comments and return 0. It would be awful if one day you wrote code and it didn't compile because the compiler it's being tested on doesn't allow for return 0 to be omitted.